core/domain/acl_decorators.py

```diff
@@ -438,6 +438,34 @@ def test_can_edit(self, exploration_id, **kwargs):
     return test_can_edit
 
 
+def can_translate_exploration(handler):
```

Comment:
This will be used for translator's controllers (for uploading audio, making audio related changes in exploration). This is similar to `can_edit_exploration` which is used in `ExplorationHandler` of editor.py